ci/experimental/set_base_image.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
```

Comment:
We could get around copying this by moving the `ci/experimental/docker` to `ci/docker` as I mentioned already